hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsRestOperation.java

```diff
@@ -311,6 +319,14 @@ private boolean executeHttpOperation(final int retryCount,
         LOG.debug("HttpRequestFailure: {}, {}", httpOperation, ex);
       }
 
+      if (ex instanceof SocketException && CONNECTION_RESET.equals(
```

Comment:
Please add comments around the need for this change and how are we enabling that.

Reply:
Added. No hooks to enable / disable.